extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mqtt" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="topic" extensible="text,number,percentage,onoff">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="broker" />
+			<bridge-type-ref id="systemBroker" />
+		</supported-bridge-type-refs>
+		<label>A topic</label>
+		<description>This thing represents a topic</description>
+	</thing-type>
+
+	<channel-type id="text">
+		<item-type>String</item-type>
+		<label>Text value</label>
+		<config-description>
+			<parameter name="stateTopic" type="text">
+				<label>Mqtt state topic</label>
```

Comment:
Add this comment only once, but will be true for all other occurrences: It should be written MQTT in labels and descriptions.